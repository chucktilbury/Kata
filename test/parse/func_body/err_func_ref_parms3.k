;-------------------------------------------------------------------------------
;	function reference with imbalanced parens
;-------------------------------------------------------------------------------

func virtual name1()() {

	namew2(()
}
