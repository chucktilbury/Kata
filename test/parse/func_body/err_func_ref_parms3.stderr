Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms3.k: 8: 2: expected a ',' or a ')' but got }
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms3.k: 9: 1: expected an optional list of expressions but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms3.k: 9: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms3.k: 9: 1: expected function body but got end of file
